[bug] Crashes on Windows due to ffprobe resolution parsing

Need to strip out `\r` for Windows new lines.

```
PS C:\Users\techy\Videos> sf6vid censor -i '.\vlc-record-2023-09-15-21h56m02s-Street Fighter 6 2023.09.12 - 15.05.24.83.DVR.mp4-.mp4' -o win-output.mp4 --p1
panic: strconv.Atoi: parsing "1080\r": invalid syntax

goroutine 1 [running]:
github.com/techygrrrl/sf6vid/video_utils.GetVideoResolution({0xc00001e0c0?, 0x0?})
        C:/Users/techy/go/pkg/mod/github.com/techygrrrl/sf6vid@v0.0.0-20230916042644-c5c8f7612f0d/video_utils/ffprobe.go:42 +0x2dc
github.com/techygrrrl/sf6vid/cmd.runCensorCmd(0xbdd020?, {0xa88bc2?, 0x5?, 0x5?})
        C:/Users/techy/go/pkg/mod/github.com/techygrrrl/sf6vid@v0.0.0-20230916042644-c5c8f7612f0d/cmd/censor.go:66 +0x70
github.com/spf13/cobra.(*Command).execute(0xbdd020, {0xc0000700a0, 0x5, 0x5})
        C:/Users/techy/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0xbdd300)
        C:/Users/techy/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        C:/Users/techy/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/techygrrrl/sf6vid/cmd.Execute()
        C:/Users/techy/go/pkg/mod/github.com/techygrrrl/sf6vid@v0.0.0-20230916042644-c5c8f7612f0d/cmd/root.go:16 +0x2e
main.main()
        C:/Users/techy/go/pkg/mod/github.com/techygrrrl/sf6vid@v0.0.0-20230916042644-c5c8f7612f0d/main.go:6 +0x17
```

![image](https://github.com/techygrrrl/sf6vid/assets/88961088/877bcdd0-5418-4fe9-936b-aa697fc497cb)

```
ffprobe -v error -select_streams v:0 -show_entries stream=width,height -of csv=s=x:p=0 '.\input.mp4-.mp4'
1920x1080
```

--- a/video_utils/ffprobe.go
+++ b/video_utils/ffprobe.go
@@ -21,9 +21,15 @@ func GetVideoResolution(path string) (*VideoResolution, error) {
 	}
 
 	// Resulting output is something like this: 1080x720
-	resolutionStr := strings.ReplaceAll(
-		string(resolutionBytes),
+	resolutionStr := string(resolutionBytes)
+	resolutionStr = strings.ReplaceAll(
+		resolutionStr,
 		"\n",
+		"",
+	)
+	resolutionStr = strings.ReplaceAll(
+		resolutionStr,
+		"\r",
 		"",
 	)
 

